src/test/java/ca/corefacility/bioinformatics/irida/ria/unit/web/services/UIProjectSampleServiceTest.java

```diff
@@ -82,8 +82,8 @@ public void testValidateNewSampleName() {
 
 	@Test
 	public void testCreateSample() {
-		CreateSampleRequest request = new CreateSampleRequest(GOOD_NAME, null);
-		ResponseEntity<AjaxResponse> response = service.createSample(request, PROJECT_1_ID, Locale.ENGLISH);
+		CreateSampleRequest[] requests = { new CreateSampleRequest(GOOD_NAME, null) };
```

Comment:
Can you also add a checks for the organism and description? Both with values and also not with values

Reply:
Please see 60e4becc57d5fd5b4f9b339ffcec2fd943f045e9.

Reply:
Sound good! Thanks!